consensus/clique/src/test/java/tech/pegasys/pantheon/consensus/clique/blockcreation/CliqueBlockCreatorTest.java

```diff
@@ -141,7 +142,8 @@ public void insertsValidVoteIntoConstructedBlock() {
             blockchain.getChainHeadHeader());
 
     final Block createdBlock = blockCreator.createBlock(0L);
-    assertThat(createdBlock.getHeader().getNonce()).isEqualTo(VoteType.ADD.getNonceValue());
+    assertThat(createdBlock.getHeader().getNonce())
+        .isEqualTo(CliqueVotingBlockInterface.voteToValue.get(VoteType.ADD));
```

Comment:
This probably needs  `CliqueVotingBlockInterface.ADD_NONCE` that it can reference rather than having to reach into `voteToValue`.

Reply:
Done.